main.js

```diff
@@ -0,0 +1,27 @@
+var titleInput = document.querySelector('#title');
+var bodyInput = document.querySelector('#body');
+var saveButton = document.querySelector('button');
```

Comment:
We noticed this isn't quite right, but we're going to fix via merge conflict editor.